docs/section10/10-5.md

```diff
@@ -0,0 +1,35 @@
+# Airside systems – Rule 10-5  
```

Comment:
ARCHIVED per discussion regarding rules that compared U_RMR to design documents and then compare U_RMR to P_RMR. This rule has been moved to the General Rules Workbook to be addressed by a master rule.